README.md

```diff
@@ -89,24 +89,24 @@ Try saying e.g. `air bat cap` to insert abc.
 
 All key commands are defined in [keys.talon](core/keys/keys.talon). Say letters of the [Talon alphabet](#alphabet) for A–Z.
 
-For modifier keys, say `help modifiers`. For example, say `shift air` to press `shift-a`, which types a capital `A`.
+For modifier keys, say `help modifiers`. For example, say `shift air` to press <kbd><kbd>shift</kbd>+<kbd>a</kbd></kbd>, which types a capital `A`.
 
 For symbols, say `help symbols`. These are defined in keys.py;
 search for `modifier_keys` and then keep scrolling — roughly starting [here](core/keys/keys.py#L124).
 
 On Windows, try commands such as:
 
-- `control air` to press Control+A and select all.
+- `control air` to press <kbd><kbd>Control</kbd>+<kbd>A</kbd></kbd> and select all.
 
-- `super-shift-sun` to press Win+Shift+S, triggering the screenshot application (Windows 10). Then try `escape` to exit.
+- `super-shift-sun` to press <kbd><kbd>Win</kbd>+<kbd>Shift</kbd>+<kbd>S</kbd></kbd>, triggering the screenshot application (Windows 10). Then try `escape` to exit.
 
 On Mac, try commands such as:
 
-- `command air` to press ⌘A and select all.
+- `command air` to press <kbd><kbd>⌘</kbd>+<kbd>A</kbd></kbd> and select all.
 
-- `control shift command 4` to press ⌃⇧⌘4, copying a screenshot of the selected area to the clipboard. Then try `escape` to exit. Please note the order of the modifiers doesn't matter.
+- `control shift command 4` to press <kbd><kbd>⌃</kbd>+<kbd>⇧</kbd>+<kbd>⌘</kbd>+<kbd>4</kbd></kbd>, copying a screenshot of the selected area to the clipboard. Then try `escape` to exit. Please note the order of the modifiers doesn't matter.
```

Comment:
```suggestion
- `control shift command 4` to press <kbd>⌃</kbd><kbd>⇧</kbd><kbd>⌘</kbd><kbd>4</kbd>, copying a screenshot of the selected area to the clipboard. Then try `escape` to exit. Please note the order of the modifiers doesn't matter.
```

macOS convention doesn't use + to denote key combinations. Regardless, the output of nested `<kbd>` elements are hard to read. Would recommend one element per key without the +'s (e.g., <kbd>⌃</kbd><kbd>⇧</kbd><kbd>⌘</kbd><kbd>4</kbd></kbd>).